tests/schema/types/test_decimal.py

```diff
@@ -31,3 +33,40 @@ def example_decimal(self, decimal: Decimal) -> Decimal:
 
     assert not result.errors
     assert result.data["exampleDecimal"] == "3.14"
+
+
+def test_serialization_of_incorrect_decimal_string():
+    """
+    Test GraphQLError is raised for an invalid Decimal.
+    The error should exclude "original_error".
+    """
+
+    @strawberry.type
+    class Query:
+        ok: bool
+
+    @strawberry.type
+    class Mutation:
+        @strawberry.mutation
+        def decimal_input(self, decimal_input: Decimal) -> Decimal:
+            assert isinstance(decimal_input, Decimal)
```

Comment:
Does this assert end up getting hit in this test?

Reply:
(Question also applies to the UUID test)

Reply:
Removed